### ISSUE TITLE
Error during onPush job execution

### ISSUE THREAD

**Issue by sandinosanchez** [qaprosoft/qps-pipeline#848]
The code delivered in #843 was a mistake, as of now we can't run successfully the onPushJob.

Logs from jenkins onPushJob

```
Also:   hudson.remoting.Channel$CallSiteStackTrace: Remote call to JNLP4-connect connection from jenkins-slave.infra/172.21.0.4:41700
		at hudson.remoting.Channel.attachCallSiteStackTrace(Channel.java:1788)
		at hudson.remoting.UserRequest$ExceptionResponse.retrieve(UserRequest.java:356)
		at hudson.remoting.Channel.call(Channel.java:998)
		at hudson.FilePath.act(FilePath.java:1069)
		at hudson.FilePath.act(FilePath.java:1058)
		at hudson.FilePath.list(FilePath.java:1896)
		at hudson.FilePath.list(FilePath.java:1880)
		at org.jenkinsci.plugins.pipeline.utility.steps.fs.FindFilesStepExecution.run(FindFilesStepExecution.java:63)
		at org.jenkinsci.plugins.pipeline.utility.steps.fs.FindFilesStepExecution.run(FindFilesStepExecution.java:44)
		at org.jenkinsci.plugins.workflow.steps.SynchronousNonBlockingStepExecution.lambda$start$0(SynchronousNonBlockingStepExecution.java:47)
		at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
		at java.util.concurrent.FutureTask.run(FutureTask.java:266)
		at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
		at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
java.io.IOException: /opt/jenkins/workspace/qaprosoft/carina-demo/onPush-carina-demo does not exist.
	at hudson.FilePath.glob(FilePath.java:1936)
	at hudson.FilePath.access$3200(FilePath.java:211)
	at hudson.FilePath$ListGlob.invoke(FilePath.java:1910)
	at hudson.FilePath$ListGlob.invoke(FilePath.java:1898)
	at hudson.FilePath$FileCallableWrapper.call(FilePath.java:3073)
	at hudson.remoting.UserRequest.perform(UserRequest.java:211)
	at hudson.remoting.UserRequest.perform(UserRequest.java:54)
	at hudson.remoting.Request$2.run(Request.java:369)
	at hudson.remoting.InterceptingExecutorService$1.call(InterceptingExecutorService.java:72)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at hudson.remoting.Engine$1.lambda$newThread$0(Engine.java:117)
	at java.lang.Thread.run(Thread.java:748)
Finished: FAILURE
```

**Comment by sandinosanchez** [comment]
@okamara fixed delivered in #849 (wait for merge into develop for testing)

Environment:

 - jenkins-master:5.1-beta4
 - infra-branch > develop
 - pipeline-branch > develop
 - sonarqube:2.2-beta1

Objective: 

 - Verify that the onPush job executes successfully

Test scenario:

 1. Run registerOrganization
 2. Run registerRepository with:

    a. pipelineLibrary > QPS-Pipeline
    b. runnerClass > com.qaprosoft.jenkins.pipeline.runner.maven.TestNG
    c. repo > carina-demo

Expected results:

 2. onPush job executed successfully

**Comment by okamara** [comment]
Tested. Fixed. 
Test env. 
jenkins-master:5.1-beta6
infra-branch > develop
pipeline-branch > develop
sonarqube:2.2-beta1
